openquakeplatform/templates/base.html

```diff
@@ -6,13 +6,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1"> 
     <title>Openquake platform {% if resource.title %}- {{ resource.title }}{% endif %}</title>
-    <meta property="og:title" content="Openquake platform {% if resource.title %}- {{ resource.title }}{% endif %}" />
-    <meta property="og:site_name" content="platform.openquake.org" />
-    <meta property="og:url" content="http://{{ SITE_DOMAIN }}{{ resource.detail_url }}" />
-    <meta property="og:description" content="{% if resource.description %}{{ resource.description  }}{% else %}Welcome to the OpenQuake Platform, by the GEM Foundation.{% endif %}" />
-    <meta name="og:image" content="{{ resource.thumbnail_url }}" />
-    <meta name="og:image:width" content="200" />
-    <meta name="og:image:height" content="150" />
+    <meta property="og:title" content="OpenQuake Platform {% if resource.title %}- {{ resource.title }}{% endif %}" />
+    <meta property="og:site_name" content="OpenQuake Platform" />
+    <meta property="og:url" content="http://platform-staging.openquake.org" />
+    <meta property="og:description" content="Welcome to the OpenQuake Platform, by the GEM Foundation." />
+    <meta property="og:image" content="https://storage-scw1.globalquakemodel.org/public/Logos/oq-logo.png" />
```

Comment:
I would bring this logo into static to reduce risk of breaking something in the future